index.js

```diff
@@ -1,5 +1,3 @@
-'use strict'
```

Comment:
looks like this was removed from other files as well. can you explain why? otherwise please add back